src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -55,16 +55,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $clearers = array();
         $container = $this->getContainer();
         $cacheDir = $container->getParameter('kernel.cache_dir');
+        $defaultClearer = $container->get('cache.default_clearer');
```

Comment:
you can't use cache.default_clearer here because some cache pool can have a different clearer, so these would be unfetcheable from this clearer.
I propose to add a new `cache.global_clearer` clearer that is unconditionally bound to all pools in CachePoolClearerPass